style.css

```diff
@@ -20,32 +20,70 @@ body {
   margin: 0;
   display: flex;
   flex-wrap: wrap;
+  /* Making this fullscreen smhow?. */
+  justify-content: left;
+  align-items: top;
+}
 
-  /* This centers our sketch horizontally. */
-  justify-content: center;
+#floating-button-area{
+  position: absolute;
+  left: 40px;
+  top: 60px;
+  width: 100px;
+  height: 200px;
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  flex-direction: column;
+}
 
-  /* This centers our sketch vertically. */
-  align-items: center;
+#key-selection-area {
+  position:absolute;
+  left:50px;
+  bottom: 50px;
+  width:1600px;
+  height:50px;
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  padding: 50px 0;
 }
 
 #sketch-canvas {
    display: block;
+   position: absolute;
+   top:0;
+   right:0;
+   bottom:0;
+   left:100px;
 }
 
-.spaced {
-   margin: 16px;
-}
-
+  /* trying some code for brush buttons */
 .toolbox__item {
-    font-size: 14px;
+    position: relative;
+    height: 42px;
+    width: 42px;
+    border-radius: 40px;
+    text-align: center;
+    line-height: 42px;
+    background: #9566DF;
+    margin-bottom: 42px;
     cursor: pointer;
-    padding: 16px;
-    border: none;
-    background: #352f44;
-    color: #ccc;
+    transition: all .25s ease;
 }
 
+
+.toolbox__item:hover {
+      box-shadow: 4px 10px 14px 0px rgba(0,0,0,0.15);
+      }
```

Comment:
indentation issue. 4 spaces before box-shadow instead of 8. `}` should be shifted to the beginning of the line